### PR TITLE
1cc order draft

### PR DESCRIPTION
**Pull request by ChetanGN**

### CODE REVIEW

File: includes/api/order.php [hunk 1]
```diff
@@ -95,6 +95,12 @@ function createWcOrder(WP_REST_Request $request)
         if (is_wp_error($orderId)) {
             $checkout_error = $orderId->get_error_message();
         }
+        //Keep order in draft status untill customer info available
+        wp_update_post(array(
```
Comment (varun-315) [inline includes/api/order.php:99]: Can we add this status in the initial create order? Any new db update may add lot of delay for slow merchants

File: woo-razorpay.php [hunk 2]
```diff
@@ -1188,7 +1193,7 @@ function check_razorpay_response()
                     $razorpayOrderId = get_transient($sessionKey);
                     $razorpayData = $api->order->fetch($razorpayOrderId);
 
-                    $this->updateOrderAddress($razorpayData['items'][0], $order);
+                    $this->updateOrderAddress($razorpayData, $order);
```
Comment (varun-315) [inline woo-razorpay.php:1196]: What is this? Looks like it is coming from a contract change?

File: woo-razorpay.php [hunk 3]
```diff
@@ -1102,6 +1102,11 @@ function check_razorpay_response()
             if (count($postIds) > 0)
             {
                 $orderId = $postIds[0];
+
+                wp_update_post(array(
```
Comment (varun-315) [inline woo-razorpay.php:1106]: Consider a new function `updateOrderStatus(string orderId, string status) { return ... }

**Review by poojarazor — APPROVED** [review]
LGTM

File: includes/api/order.php (hunk 1 above)
Comment (ChetanGN) [reply includes/api/order.php:99]: During the order creation by default, wooc will set the status to wc-pending. No option is available to create it in draft status.

**Pull request merged**